lib/puppetserver/ca/generate_action.rb

```diff
@@ -1,6 +1,6 @@
 require 'optparse'
 require 'openssl'
-require 'puppetserver/utils/file_utils'
+require 'puppetserver/utils/file_utilities'
```

Comment:
these are named incorrectly currently, I should probably pull these changes into their own commit.